invokeai/frontend/web/src/features/gallery/components/ImageGalleryContent.tsx

```diff
@@ -185,11 +185,12 @@ const ImageGalleryContent = () => {
   useEffect(() => {
     dispatch(
       receivedPageOfImages({
-        categories: ['general'],
+        categories,
         is_intermediate: false,
       })
     );
-  }, [dispatch]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@psychedelicious is there a better way to run something only on mount?

Reply:
Nope that's how you do it. Could also have a conditional to check if there are already images loaded and not make the request if so.